started/demo.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: Run Fyne Demo
+
+---
+
+If you want to see the Fyne toolkit in action before you start to code your own application,
+you can see our [demo app](https://github.com/fyne-io/fyne/tree/master/cmd/fyne_demo) application.
```

Comment:
Having this worded "demo app application" seems a bit to me like it stutters. I think we should word that a bit different.